src/Sulu/Component/DocumentManager/Slugifier/PathCleanupSlugifier.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager\Slugifier;
+
+use Sulu\Component\PHPCR\PathCleanupInterface;
+use Symfony\Cmf\Api\Slugifier\SlugifierInterface;
+
+class PathCleanupSlugifier implements SlugifierInterface
+{
+    /** @var PathCleanupInterface */
+    private $pathCleanup;
+
+    public function __construct(PathCleanupInterface $pathCleanup)
+    {
+        $this->pathCleanup = $pathCleanup;
+    }
+
+    public function slugify($text)
+    {
+        return $this->pathCleanup->cleanup($text);
```

Comment:
looks like this is the reason for the `ContentRepositoryTest` errors. phpcr cannot handle `/` inside of paths. the previously used `Urlizer` class replaced these with `-`. so we need to replace `/` with `-` in this class too to keep things compatible.

Reply:
And I think our resource locator field in the form also behaves in an unexpected behavior if the editable part contains a slash.